shared/bytesutil/bytes.go

```diff
@@ -142,3 +142,11 @@ func Trunc(x []byte) []byte {
 	}
 	return x
 }
+
+// ToLowU64 returns the lowest 8 bytes interpreted as little endian.
+func ToLowU64(x []byte) int64 {
```

Comment:
Please rename to `ToLowInt64` as it's not unsigned anymore

Reply:
done @prestonvanloon